jakelib/utils.js

```diff
@@ -83,6 +83,26 @@ const replacer = (value, variables) => {
   return value;
 }
 
+const CLUSTERS_PROTECTION_STATUS = {
+    prod: {
+        lv : { protected: true },
+    },
+    stag: {
+        pp: { protected: true },
+        one: { protected: false },
+        two: { protected: false },
+        qe: { protected: false },
+        uat: { protected: false },
+    },
```

Comment:
i got rid of `eph` from this object, i didn't see it being used at all

Reply:
@tom-con maybe you know better

Reply:
@aprakash-gc won't that stop people from being able to deploy to ephemeral environments?